processing/src/main/java/io/druid/query/filter/JavaScriptDimFilter.java

```diff
@@ -21,32 +21,36 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Joiner;
 import com.google.common.base.Preconditions;
 import com.metamx.common.StringUtils;
 
 import java.nio.ByteBuffer;
+import java.util.Arrays;
 
 public class JavaScriptDimFilter implements DimFilter
 {
-  private final String dimension;
+  private final String[] dimensions;
   private final String function;
 
   @JsonCreator
   public JavaScriptDimFilter(
```

Comment:
How about to make two constructor? But his will need opinions from others because I don't know well on guava.